### PR TITLE
Update nodepool and zuul

### PR DESCRIPTION
**Pull request by h00130372**

### CODE REVIEW

File: config/zuul/openlab-main.yaml [hunk 1]
```diff
@@ -13,32 +13,22 @@
...
+          - gophercloud/gophercloud:
+              exclude-unprotected-branches: true
+          - terraform-providers/terraform-provider-openstack:
+              exclude-unprotected-branches: true
           - flystack/misty
           - fog/fog-openstack
           - dtantsur/rust-openstack
           - kubernetes/cloud-provider-openstack:
               exclude-unprotected-branches: true
-          - theopenlab/ansible:
```
Comment (kiwik) [inline config/zuul/openlab-main.yaml]: should be `ansible/ansible`, and the following project should be changed.

File: config/zuul/openlab-main.yaml [hunk 2]
```diff
@@ -13,32 +13,22 @@
...
-              exclude-unprotected-branches: true
-          - zhongjun2/bosh-huaweicloud-cpi-release:
-              exclude-unprotected-branches: false
-          - zhongjun2/cf-huaweicloud-validator:
-              exclude-unprotected-branches: true
-          - liu-sheng/bosh-acceptance-tests:
-              exclude-unprotected-branches: true
           - huaweicloud/huaweicloud-service-broker:
               exclude-unprotected-branches: true
-          - openservicebrokerapi/osb-checker
```
Comment (kiwik) [inline config/zuul/openlab-main.yaml]: why delete this project?

File: config/zuul/openlab-main.yaml [hunk 3]
```diff
@@ -13,32 +13,22 @@
...
           # Offical
           # NOTE gophercloud and terraform master are not
           # protected branches
-          - gophercloud/gophercloud
-          - terraform-providers/terraform-provider-openstack
+          - gophercloud/gophercloud:
+              exclude-unprotected-branches: true
+          - terraform-providers/terraform-provider-openstack:
+              exclude-unprotected-branches: true
           - flystack/misty
```
Comment (kiwik) [inline config/zuul/openlab-main.yaml]: I prefer to format all of project in this schema:
```
          - gophercloud/gophercloud:
              exclude-unprotected-branches: true/false
```

File: config/zuul/openlab-main.yaml (hunk 2 above)
Comment (h00130372) [reply config/zuul/openlab-main.yaml]: made a mistake when paste/copy

**Pull request closed without merging**